packit/utils.py

```diff
@@ -128,7 +126,11 @@ def run_command(
         cmd,
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
-        shell=False,
+        # we know what we are doing here:
+        #  * we already run commands defined by users - it's gonna be more convenient for them
+        #    to do `ls *.tar.gz` instead of `bash -c "ls *.tar.gz"`
```

Comment:
So basically this is done to have globing? Or is there also some other reason?

If it's only for globing wouldn't be better to use [`glob.glob()`](https://docs.python.org/3.8/library/glob.html#glob.glob) instead of a shell?

Reply:
Probably also env. vars and command chaining?

Reply:
> So basically this is done to have globing? Or is there also some other reason?
> 
> If it's only for globing wouldn't be better to use [`glob.glob()`](https://docs.python.org/3.8/library/glob.html#glob.glob) instead of a shell?

not just globbing, all shell syntax:
* `ls | grep asd`
* `grep $(find)`
* ...